nextjs.js

```
@@ -0,0 +1,44 @@
+    return { __html: "<script>alert('XSS!')</script>" };
+  }
+
+  return (
+    <div>
+      <h1>Next.js Page</h1>
+      
+      <div style={{ color: "red" }}>This is an inline-styled div</div>
+      
+      <div dangerouslySetInnerHTML={createMarkup()} />
+      
+      <button onClick={() => { setCount(count + 1); setCount(count + 2); }}>
```

Biome

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)